doc/effects.md

```diff
@@ -224,6 +225,26 @@ tree after the specified delay has passed:
 final effect = RemoveEffect(delay: 10.0);
 ```
 
+## ColorEffect
+
+This effect will change the base color of the paint, causing the rendered component to be tinted by
+the provided color between a provided range.
```

Comment:
Don't forget to mention that this effect doesn't mix if several of these effects are added to the same component.